src/main/java/com/airlines/entity/Airplane.java

```diff
@@ -0,0 +1,87 @@
+package com.airlines.entity;
+
+import java.time.LocalDate;
+
+public class Airplane {
+    private final int id;
+    private final String codeName;
+    private final String model;
+    private final LocalDate manufacture;
+    private final int capacity;
+    private final int flightRange;
+    private final int crews_id;
```

Comment:
Java naming convention for field name is CamelCase.
So crews_id should be crewId.

And pay attentions:
crews - it's plural because it's database name.
crew - entity

so crewId

Reply:
fixed in 8d230f9c3c3300d7da8f899a28eb4137f3bf8074